### ISSUE TITLE
Bridge.begin hangs

### ISSUE THREAD

**Issue by davehun** [domino-team/domino-firmware#3]
All sketches that include the yun bridge function hang during setup at
Bridge.begin();

e.g
 connect to the console and run
# include <Bridge.h>
# include <YunServer.h>
# include <YunClient.h>

YunServer server;

void setup() {
  // put your setup code here, to run once:
  Serial.begin(9600); //Start the serial connection with the computer

  Serial.println("starting bridge");
  Bridge.begin();
  Serial.println("Bridge started");

starting bridge wiill be printed to the serial port but bridge started never will be

**Comment by davehun** [comment]
it appears the run-bridge command is not starting / running correctly

**Comment by alzhao** [comment]
@davehun, you are right. You can use Arduino firmware to use the bridge for now. We will check and update the firmware. The instructions are here: https://github.com/domino-team/docs/blob/master/qi/arduino-yun.md or http://domino.io/docs/qi/arduino-yun.html
